<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MD5</class>
 <widget class="QWidget" name="MD5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>254</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MD5加密</string>
  </property>
  <widget class="QGroupBox" name="groupBoxMD5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 27, 27);</string>
   </property>
   <property name="title">
    <string>MD5加密</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="labelInputString">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文中宋&quot;;</string>
    </property>
    <property name="text">
     <string>要加密的字符串:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInput">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="placeholderText">
     <string>请出入待加密的字符串！</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMD5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 75 11pt &quot;微软雅黑&quot;;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelString">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>字符串</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditString">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label16min">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16位 小写</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit16min"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label16max">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16位 大写</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit16max"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label32min">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>32位 小写</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit32min"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label32max">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>32位 大写</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit32max"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
